src/ucp/wireup/wireup_cm.c

```diff
@@ -707,6 +711,7 @@ static unsigned ucp_ep_cm_disconnect_progress(void *arg)
     ucp_ep_h ucp_ep            = arg;
     uct_ep_h uct_cm_ep         = ucp_ep_get_cm_uct_ep(ucp_ep);
     ucs_async_context_t *async = &ucp_ep->worker->async;
+    int invoke_err_cb          = 0;
```

Comment:
since this is an out arg of `ucp_ep_cm_remote_disconnect_progress` it should be initialized there